server/channels/api4/system.go

```diff
@@ -194,6 +195,13 @@ func getSystemPing(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	s["ActiveSearchBackend"] = c.App.ActiveSearchBackend()
 
+	// Checking if mattermost is running as root
+	rootStatusKey := "root_status"
+	s[rootStatusKey] = "false"
+	if os.Geteuid() == 0 {
+		s[rootStatusKey] = "true"
+	}
```

Comment:
Using `strconv.FormatBool` we can simplify that a bit more.
```suggestion
	// Checking if mattermost is running as root
	s["root_status"] = strconv.FormatBool(os.Geteuid() == 0)
```